packages/react/src/PageMenu/PageMenu.tsx

```diff
@@ -45,16 +45,18 @@ export const PageMenu = forwardRef(
 
 export interface PageMenuLinkProps extends PropsWithChildren<AnchorHTMLAttributes<HTMLAnchorElement>> {
   icon?: Function
+  showOnMobile?: boolean
```

Comment:
We mustn’t use the word ‘mobile’ and think mobile-first. Maybe something like `hideInNarrowGrid`?

Reply:
Since we've decided the menu button should be placed in the megamenu, and this solution is only to hide everything but the menu button i'll go ahead and remove this and the menu button from page menu.